components/Avatar/Avatar.css

```diff
@@ -0,0 +1,29 @@
+.img {
+  width: 3.6rem;
+  height: 3.8rem;
+  border: 1.1px solid black;
+  margin: 2rem 1.5rem;
+}
```

Comment:
```suggestion
    object-fit: cover;
}
```

Reply:
Refer this:
[MDN Object Fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)